### ISSUE TITLE
Checkmarks should be on the left side of challenges on curriculum modal

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/learn#103]
Here's what 

learn.freecodecamp.org currently looks like: 

![basic_html_and_html5___freecodecamp](https://user-images.githubusercontent.com/985197/40580813-ba5f51fc-6179-11e8-993a-03354a5962e2.png)

And here's what we want it to look like based on our old map:

![delete_html_elements___freecodecamp](https://user-images.githubusercontent.com/985197/40580808-9e95dbd0-6179-11e8-9f25-e00d1f78ff7f.png)

**Comment by tchaffee** [comment]
I can take this on if no one has yet.

**Comment by tchaffee** [comment]
I started to look at this and I think in order to get the checkmarks to appear I need to be logged in? When I try to log in it takes me to https://auth.freecodecamp.org which then redirects me to https://beta.freecodecamp.org/ when I enter the authorization code. How can I authenticate locally when testing?

**Comment by tchaffee** [comment]
I opened a new issue https://github.com/freeCodeCamp/learn/issues/105.

**Comment by tchaffee** [comment]
@raisedadead You can unblock this. I found a way to test it without being logged in. I'll have a PR soon.

**Comment by raisedadead** [comment]
@tchaffee you are amazing! Can you please share the workaround for benefit of others? And thanks a lot for working on this.

**Comment by tchaffee** [comment]
@raisedadead The auth workaround is underwhelming and might not be so useful to others. But you can see it in this comment on the PR. https://github.com/freeCodeCamp/learn/pull/106#issuecomment-392590932
